### PR TITLE
Feature: adding Dilation to the filtering operations

### LINKED ISSUE #20: Feature: adding Dilation to the filtering operations
**Erosion** and **dilation** are the two types of morphological operations. As the name implies, morphological operations are the set of operations that process images according to their shapes.

Based on the given input image a "structural element" is developed. This might be done in any of the two procedures. These are aimed at removing noise and settling down the imperfections, to make the image clear

You can perform the dilation operation on an image using the **dilate()** method of the **OpenCV imgproc class**.

Check the [project documentation](https://www.scorelab.org/ImageLab/) for more information related to the project setup and contributions.

### PR DESCRIPTION
**Pull request by SahanDisa**
Added  Dilation Filter for the filter operations.

Fixes #20 

dil

### CODE REVIEW

**Comment by ivantha** [comment]
Please remove the unnecessary project files from the PR.

File: src/main/java/com/imagelab/util/Constants.java [hunk 1]
```diff
@@ -51,13 +51,13 @@
...
             + "operators that you can drag and drop to the"
             + "playground area.\n\n"
-            + "Playground: area that you can use to build your"
+            + "• Playground: area that you can use to build your "
             + "openCV pipeline.\n\n"
-            + "Preview pane: area which you can see your output.\n\n"
-            + "Properties pane: where you can change the properties"
+            + "• Preview pane: area which you can see your output.\n\n"
+            + "• Properties pane: where you can change the properties "
             + "related to the operators.";
```
Comment (ivantha) [inline src/main/java/com/imagelab/util/Constants.java]: Are these relevant changes?

**Review by ivantha — CHANGES_REQUESTED** [review]

File: src/main/java/com/imagelab/util/Constants.java (hunk 1 above)
Comment (SahanDisa) [reply src/main/java/com/imagelab/util/Constants.java]: Sure I will remove those changes, it's irrelevant here

Comment (SahanDisa) [reply src/main/java/com/imagelab/util/Constants.java]: @ivantha I removed it.

**Pull request closed without merging**